### ISSUE TITLE
Sample code

### ISSUE THREAD

**Issue by heitorcolangelo** [heitorcolangelo/kappuccino#73]
It would be good if the sample code could me more organized and more complete.

**Comment by alancamargo92** [comment]
Is this enhancement still needed?

**Comment by heitorcolangelo** [comment]
Actually it is, the sample code it's really messed =/.
At the time I had to hurry with those samples and it would be really good if they are better organised.
It's open to discussion on how organise them.

**Comment by alancamargo92** [comment]
Yeah makes sense. I think it would be nice to write the sample in Kotlin as well, since the focus of the project itself is Kotlin. What do you think?

**Comment by heitorcolangelo** [comment]
Which part of the sample? The test code or the code to be tested?

**Comment by alancamargo92** [comment]
The code to be tested is written in Java

**Comment by heitorcolangelo** [comment]
Oh i got you, yeah, it's also a good idea to move the code to be tested to kotlin.
This can be easily done after organising it. 😬 
It would also be really good to have two versions of the sample code, one for kotlin and one for java.
